CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
   * Require `braintree_ios` major version v5
   * Replace `VisaCheckout.framework` v7.2.0 with `VisaCheckout.xcframework` v7.3.1
   * Drop Carthage support (Carthage support wasn't working in the latest 2 major versions)
+* Add Swift Package Manager support
```

Comment:
Is moving the header files considered a breaking change?

Reply:
We're releasing a major version, so we can do that! And honestly - I don't think it would constitute a breaking change anyways